routes/v1.js

```diff
@@ -119,6 +122,7 @@ function addRoutes(app, peliasConfig) {
       postProc.parseBoundingBox(),
       postProc.normalizeParentIds(),
       postProc.geocodeJSON(peliasConfig, base),
+      postProc.assignLabels(),
```

Comment:
it would be great if assignLabels could go first in all the endpoints, for consistency